plugin-code/src/main/resources/template/macros/scenario.ftl

```diff
@@ -105,22 +105,63 @@ limitations under the License.
 
 <#macro attachments step>
     <#if step.embeddings??>
+    	<#assign index=1>
```

Comment:
List items have an automatic index: `${x?index}` so an index variable is not needed.

Reply:
Removed this code